plugins/woocommerce/src/Internal/DataStores/Orders/LegacyDataCleanup.php

```diff
@@ -182,10 +173,13 @@ public function can_run() {
 	/**
 	 * Checks whether the cleanup process should run. That is, it must be activated and {@see can_run()} must return TRUE.
 	 *
+	 * @deprecated 9.1.0
+	 *
 	 * @return boolean TRUE if the cleanup process should be run, FALSE otherwise.
 	 */
 	public function should_run() {
-		return $this->can_run() && $this->is_flag_set();
+		wc_deprecated_function( __METHOD__, '9.1.0', __CLASS__ . '::can_run()' );
```

Comment:
We are in the `Internal` namespace. We don't need to use the regular deprecation procedure, if we don't need the method anymore we can just remove it.

Reply:
I've been bitten by this before but I guess this isn't a widely used class so we're good. I've removed the methd in 485bd7bfc6982dd67507d6a185ea0b39ca3373e7.